Sources/Plasma/PubUtilLib/plPipeline/plTextFont.cpp

```diff
@@ -132,6 +129,10 @@ uint16_t  *plTextFont::IInitFontTexture()
 
     DeleteDC( hDC );
     DeleteObject( hFont );
+    
+    int iScale = int( ceil( GetDeviceCaps(hDC, LOGPIXELSY) / 96.0f ) );
```

Comment:
Maybe use `plWinDpi` for this?

Reply:
I decided to just use fFontHeight instead which should vary based on the DPI. I was worried my change would introduce non-power-of-two textures which is not a problem in Metal but might be on older Direct3D hardware. Not sure about the status of non power of two in Direct3D 9.